SECP-service/src/main/resources/assets/app/js/password/password-controller.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+angular.module('SECP')
+    .controller('PasswordController', ['$scope', '$window', function ($scope, $window) {
+        $scope.sendEmail = function () {
+            $window.alert("email sent");
```

Comment:
Please use a SWAL here. There are lots of examples sprinkled in the controllers.